client-course-schedulizer/src/components/Tabs/FacultySchedule/FacultySchedule.scss

```diff
@@ -0,0 +1,14 @@
+.calendars-wrapper {
+  overflow-x: auto;
+}
+
+.calendars-adjacent {
+  width: fit-content;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+}
+
+.calendar {
+  width: 600px; // This is static, and I don't like it, but I don't know how to make it better
+}
```

Comment:
Would it be possible to make this smaller to fit another professor on the screen?

Reply:
Also, is it possible to make the day labels "M", "T", "W", "Th", and "F"?

Reply:
`25vw` seems to work for me other than the overlapping day headings. Sometimes it's weird right after I change the scss file, but if I reload the page it's adjusts nicely.

Reply:
Good call, I didn't try reloading the page. It works great! Thanks.